src/ui/src/pages/app/monitors/Index/components/MonitorsTable/index.tsx

```diff
@@ -38,11 +38,12 @@ const MonitorsTable: React.FC<{
       dataField: 'metrics',
       text: "Status",
       formatter: (cell: any, row: any) => {
-        if (row.timestamp_field === null) {
-                return (
-                  <span className="badge rounded-pill bg-danger">Disabled</span>
-                )
-        } else if (row.metrics == 0) {
+        // if (row.timestamp_field === null) {
+        //         return (
+        //           <span className="badge rounded-pill bg-danger">Disabled</span>
+        //         )
+        // } else 
```

Comment:
Let's just remove this if it will never be `null` now?

Reply:
the timestamp field logic is still going to apply in the near future I would imagine